data/seriesRepository.go

```diff
@@ -313,7 +314,8 @@ func (r *SeriesRepository) GetSeriesSiblingsFreqById(
 ) (frequencyList []models.FrequencyResult, err error) {
 	rows, err := r.DB.Query(`SELECT DISTINCT(RIGHT(series.name, 1)) as freq
 	FROM series JOIN (SELECT name FROM series where id = ?) as original_series
-	WHERE series.name LIKE CONCAT(left(original_series.name, locate("@", original_series.name)), '%');`, seriesId)
+	WHERE series.name LIKE CONCAT(left(original_series.name, locate("@", original_series.name)), '%')
+	ORDER BY FIELD(freq, "A", "S", "Q", "M", "W", "D");`, seriesId)
```

Comment:
I know SQL Server has inline tables so you can create sort orders on the fly that way.